lua/goplements/init.lua

```diff
@@ -148,20 +148,34 @@ M.implementation_callback = function(fcache, result, publish_names)
   publish_names(names)
 end
 
+--- Utility function for using deprecated function
+---@param client vim.lsp.Client The LSP client
+---@param params table
+---@param callback function
+local function request_implementation(client, params, callback)
+  if vim.fn.has("nvim-0.11") == 1 then
```

Comment:
Thanks! Probably better to check `vim.version()` for the version instead of the has
Then if anyone is on nvim 0.10 still it won't break

Something like:

```suggestion
  if vim.version.minor <= 11 then
```

Reply:
I was wrong :D